tensorboard/loader_test.py

```diff
@@ -228,7 +228,10 @@ def testBigNumberInAmerica_showsCommas(self):
     self.assertIn(u' 1,024B/s ', self.bars[0])
 
   def testBigNumberInGermany_showsCommas(self):
-    locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    try:
+      locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    except Exception:
```

Comment:
Wait, why did this raise a lint error [here](https://travis-ci.org/tensorflow/tensorboard/jobs/258297196) but not [here](https://travis-ci.org/tensorflow/tensorboard/jobs/258355256)? Both have this line.